daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Validation.scala

```diff
@@ -13,20 +13,20 @@ private final class Validation[Nid, Cid]() {
     *   the root nodes of [[recorded]].
     * @note This function is symmetric
     *
-    * 'isReplayedBy' normalizes its 2nd argument, and then determines structural equality.
+    * 'isReplayedBy' is simply structural equality.
```

Comment:
I’m confused, how can that be true? Where do you normalize for example `byKey`?

Reply:
Yes. This has  been lost.
I mean to directly build the correct normalized value for byKey.

Reply:
AHA. Explains this failure on CI (only Linux) ?
```
//ledger/ledger-api-test-tool:conformance-test-1.8
```

Reply:
yep seems very likely

Reply:
I fixed the `byKey` normalization. This test passes again.